doc/conf.py

```diff
@@ -62,6 +62,9 @@
     "sphinx_gallery.gen_gallery",
     "sphinxcontrib.cairosvgconverter",
 ]
+needs_extensions = {
+    "sphinx_gallery.gen_gallery": "0.19.0",
```

Comment:
> The version strings should be in the 'major.minor' form.

Remove the patch version part?

```suggestion
    "sphinx_gallery.gen_gallery": "0.19",
```